src/server/server.ts

```diff
@@ -185,11 +245,17 @@ async function serveArticlePost(
 		const renderingRequest = await mapiDecoder(req.body);
 		const richLinkDetails = req.query.richlink === '';
 		const isEditions = req.query.editions === '';
+		const themeOverride = themeFromUnknown(req.query.theme);
 
 		if (richLinkDetails) {
 			void serveRichLinkDetails(renderingRequest, res);
 		} else {
-			void serveArticle(renderingRequest, res, isEditions);
+			void serveArticleSwitch(
+				renderingRequest,
+				res,
+				isEditions,
+				themeOverride,
+			);
```

Comment:
Is this needed in `serveArticlePost`? This function is never triggered for Editions is it? Editions uses `serveEditionsArticlePost`?

Reply:
Good point, fixed!